portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* Create a new HTML element */
+function createElementWithParams (tag, {
+  className = "",
+  innerText = "",
+  event = undefined
```

Comment:
Since we assume this is a click handler, it may be clearer to mention that in the name. e.g. onClickHandler.